runtime/balrt.h

```diff
@@ -108,9 +108,20 @@ typedef GC struct Mapping {
     uint8_t tableLengthShift;
 } *MappingPtr;
 
+typedef struct {
+    uintptr_t pc;
+    // Index into the second array of start of string
+    // string is terminated with zero
+    unsigned fileOffset;
```

Comment:
can't we use uint*_t style here?

Reply:
It’s going to be uint64_t on the platforms we support, but I think it’s clearer to use uintptr_t since that’s what libbacktrace uses.

Reply:
`uintptr_t` is fine, I was commenting on the `unsigned`.

Reply:
Oh, I see. Yes, uint32_t would be better here.